k8s/overlays/nerc-shift-1/clients/ocp-prod.yaml

```diff
@@ -0,0 +1,106 @@
+      display.on.consent.screen: "false"
+      exclude.session.state.from.auth.response: "false"
+      id.token.as.detached.signature: "false"
+      oauth2.device.authorization.grant.enabled: "false"
+      oidc.ciba.grant.enabled: "false"
+      require.pushed.authorization.requests: "false"
+      saml.artifact.binding: "false"
+      saml.assertion.signature: "false"
+      saml.authnstatement: "false"
+      saml.client.signature: "false"
+      saml.encrypt: "false"
+      saml.force.post.binding: "false"
+      saml.multivalued.roles: "false"
+      saml.onetimeuse.condition: "false"
+      saml.server.signature: "false"
+      saml.server.signature.keyinfo.ext: "false"
+      saml_force_name_id_format: "false"
+      tls.client.certificate.bound.access.tokens: "false"
+      use.refresh.tokens: "true"
+    clientAuthenticatorType: client-secret
+    clientId: ocp-prod
+    defaultClientScopes:
+    - web-origins
+    - roles
+    - profile
+    - email
+    directAccessGrantsEnabled: true
+    enabled: true
+    fullScopeAllowed: true
+    implicitFlowEnabled: false
+    nodeReRegistrationTimeout: -1
+    optionalClientScopes:
+    - address
+    - phone
+    - offline_access
+    - microprofile-jwt
+    protocol: openid-connect
+    protocolMappers:
+    - config:
+        access.token.claim: "false"
+        claim.name: cilogon_idp_name
+        id.token.claim: "true"
+        jsonType.label: String
+        user.attribute: cilogon_idp_name
+        userinfo.token.claim: "true"
+      name: cilogon_idp_name
+      protocol: openid-connect
+      protocolMapper: oidc-usermodel-attribute-mapper
+    - config:
+        access.token.claim: "false"
+        claim.name: preferred_username
+        id.token.claim: "true"
+        jsonType.label: String
+        user.attribute: username
+        userinfo.token.claim: "true"
+      name: username
+      protocol: openid-connect
+      protocolMapper: oidc-usermodel-property-mapper
+    - config:
+        access.token.claim: "true"
+        claim.name: clientHost
+        id.token.claim: "true"
+        jsonType.label: String
+        user.session.note: clientHost
+      name: Client Host
+      protocol: openid-connect
+      protocolMapper: oidc-usersessionmodel-note-mapper
+    - config:
+        access.token.claim: "true"
+        claim.name: clientAddress
+        id.token.claim: "true"
+        jsonType.label: String
+        user.session.note: clientAddress
+      name: Client IP Address
+      protocol: openid-connect
+      protocolMapper: oidc-usersessionmodel-note-mapper
+    - config:
+        access.token.claim: "true"
+        claim.name: clientId
+        id.token.claim: "true"
+        jsonType.label: String
+        user.session.note: clientId
+      name: Client ID
+      protocol: openid-connect
+      protocolMapper: oidc-usersessionmodel-note-mapper
+    publicClient: false
+    redirectUris:
+    - https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/*
```

Comment:
Might also want to include the public version of this: `https://console-openshift-console.apps.shift.nerc.mghpcc.org/*`. Does the oauth route also need to be included here (for both for internal and public views)?

Reply:
Added the public version and the oauth route.